tgstation.dme

```diff
@@ -1162,7 +1162,7 @@
 #include "code\modules\mob\living\silicon\pai\personality.dm"
 #include "code\modules\mob\living\silicon\pai\recruit.dm"
 #include "code\modules\mob\living\silicon\pai\say.dm"
-#include "code\modules\mob\living\silicon\pai\software.dm"
+#include "code\modules\mob\living\silicon\pai\software_new.dm"

```

Comment:
None in particular, no. I would like to keep it around so I can look at the Camera Jack proc and such when I get around to finishing that.